docs/guide/extending_ez_platform_ui.md

```diff
@@ -165,6 +167,250 @@ $menu->reorderChildren(
 );
 ```
 
+## Tabs
+
+Many elements of the back-office interface, such as System Info or Location View, are built using tabs.
+
+You can extend existing tab groups with new tabs, or create your own tab groups.
+
+### Adding a new tab group
+
+To create a custom tab group with additional logic you need to create it at the level of compiling the Symfony container, using a CompilerPass:
```

Comment:
This example needs some explanations IMHO.

Reply:
Could we add a link to the Sf documentation on CompilerPass?
https://symfony.com/doc/current/service_container/compiler_passes.html